src/parameterizations/lateral/MOM_MEKE.F90

```diff
@@ -81,7 +81,8 @@ module MOM_MEKE
   logical :: Rd_as_max_scale !< If true the length scale can not exceed the
                         !! first baroclinic deformation radius.
   logical :: use_old_lscale !< Use the old formula for mixing length scale.
-  logical :: use_min_lscale !< Use simple minimum for mixing length scale.
+  logical :: use_min_lscale !< Use simple minimum for mixing l >= 0.
```

Comment:
Is the change to the comment on line 84 intentional?

Reply:
No, it was not. Thanks for catching that.